versioned_docs/version-1.1.x/deployment/google.mdx

```diff
@@ -230,7 +230,7 @@ surrealdb-tikv   <none>   *       34.160.82.177   80      5m
 4. Connect to the cluster and define the initial credentials:
 
 ```bash title="Define initial credentials"
-$ export SURREALDB_URL=http://$(kubectl get ingress surrealdb-tikv -o json | jq -r .status.loadBalancer.ingress[0].ip)
+$ export SURREALDB_URL=http://$(kubectl get ingress surrealdb-tikv -o json | jq -r ".status.loadBalancer.ingress[0].ip")
```

Comment:
Good catch! thanks for the contribution

```suggestion
$ export SURREALDB_URL=http://$(kubectl get ingress surrealdb-tikv -o json | jq -r '.status.loadBalancer.ingress[0].ip')
```

In general I prefer single quotes so we avoid unintended interpolations from the shell